File: 시저 암호.py
def solution(s, n):
    answer = ''
    for i in s:
        if i == " ":
            answer += i
        elif ord(i) in range(64, 91) and ord(i)+n not in range(64, 91):
            answer += chr(ord(i) + n - 26)
        elif ord(i) in range(97, 123) and ord(i)+n not in range(97, 123):
            answer += chr(ord(i) + n - 26)
        elif i.isalpha():
            answer += chr(ord(i) + n)
    return answer

File: level_1/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == ' ') {
                answer += s.charAt(i);
                continue;
            }
            if ((s.charAt(i) <= 90 && s.charAt(i) + n > 90) ||
                    (s.charAt(i) <= 122 && s.charAt(i) + n > 122))
                answer += (char) (s.charAt(i) + n - 26);
            else
                answer += (char) (s.charAt(i) + n);
        }
        return answer;
    }
}